src/dmd/dcast.d

```diff
@@ -840,6 +840,42 @@ MATCH implicitConvTo(Expression e, Type t)
             }
         }
 
+        override void visit(CatExp e)
```

Comment:
Every new function must come with DDOC

Reply:
I've added a ddoc but to honest the whole idea of documenting this visit overload when there tens more having now doc doesn't feel right.